src/Tribe/Attendees_Table.php

```diff
@@ -956,6 +956,15 @@ public function prepare_items() {
 			];
 		}
 
+		// Setup sorting args.
+		if ( tribe_get_request_var( 'orderby' ) ) {
+			$args['orderby'] = tribe_get_request_var( 'orderby' );
+		}
+
+		if ( tribe_get_request_var( 'order' ) ) {
+			$args['order']   = tribe_get_request_var( 'order' );
+		}
```

Comment:
Could we use the default parameter instead of calling `tribe_get_request_var()` twice for each of these?
```suggestion
		$args['orderby'] = tribe_get_request_var( 'orderby', $args['orderby'] );
		$args['order']   = tribe_get_request_var( 'order', $args['order'] );
```

Reply:
@codingmusician I had to revert this again as tests were failing since `$args['orderby']` is not set always and giving error. For now, we should be good with the previous call.